### ISSUE TITLE
Some buttons are not labeled correctly for screen readers.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [transmission-remote-gui/transgui#808]
```
Buttons are not labeled correctly for screen readers, so it just says button 
for both ok/cancel or yes/no.
Would you mind labeling the buttons?
Nvaccess.org has a great free screen reader you could try with Windows.
Also if you have windows 7 or 8, you can run the narrator by pressing windows+u.

What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
transgui 5.0.1 on Windows 7 sp1


Please provide any additional information below.

```

Original issue reported on code.google.com by `seanlee...@gmail.com` on 29 Jun 2014 at 10:55

**Comment by Neurrone** [comment]
Just tried this with the NVDA screen reader and this problem is still present.
